src/devices/uFire_ISE/UFireIse.cs

```diff
@@ -14,9 +14,17 @@ namespace Iot.Device.UFire
     /// </summary>
     public class UFireIse : IDisposable
     {
+        /// <summary>
+        /// 0x3F is the default address of all sensors
+        /// </summary>
+        public static byte I2cAddress = 0x3F;
+
+        // A temperature measurement takes 750ms, see https://www.ufire.co/docs/uFire_ISE/#use
         private const int ISE_TEMP_MEASURE_TIME = 750;
```

Comment:
Thanks for adding the links. So would make sense then to adjust all the naming for the constants in this file and in the classes with the standard notation: IseTemperatureMeasureTime, IseMvMeasureTime, etc